manywheel/build_cpu.sh

```diff
@@ -208,10 +208,20 @@ cp /$WHEELHOUSE_DIR/torch*.whl /remote/$WHEELHOUSE_DIR/
 # remove stuff before testing
 rm -rf /opt/rh
 
+# The package's name that we made could be torch-nightly
+if [[ -n "$PACKAGE_NAME" ]]; then
+  package_name="$PACKAGE_NAME"
+else
+  package_name='torch'
+fi
+echo "Expecting the built wheels to be packages for '$package_name'"
+
+
+# Test that all the wheels work
 export OMP_NUM_THREADS=4 # on NUMA machines this takes too long
 pushd $PYTORCH_DIR/test
 for PYDIR in "${DESIRED_PYTHON[@]}"; do
-    "${PYDIR}/bin/pip" uninstall -y torch
-    "${PYDIR}/bin/pip" install torch --no-index -f /$WHEELHOUSE_DIR
+    "${PYDIR}/bin/pip" uninstall -y "$package_name"
```

Comment:
this is super weird that this works (or is it not uninstalling maybe?). I think to allow a configurable package_name, you need to make changes to pytorch/setup.py, around here: https://github.com/pytorch/pytorch/blob/master/setup.py#L1026

Reply:
I have those corresponding changes here https://github.com/pytorch/pytorch/pull/9544 . I'm using that PR to test changes in the pytorch repo

Reply:
The environment variable that this expects PACKAGE_NAME is also written into the CI jenkins code